### DISCUSSION TITLE
Displayed feeds

### DISCUSSION THREAD

**Discussion by kvten** [osmoscraft/osmosfeed#85] [Q&A]
Awesome project! Have these questions:

1. What is the current limit on how many feeds are displayed?
2. Is there currently an option to limit number of displayed feeds?

**Comment by chuanqisun** [comment]
Hey kenf1, there is no limit right now.

Assuming you want to limit the number of feeds that are displayed, how would you like to do so? e.g. Do you want to limit them during build time? Or dynamically on the UI? And how would you like to limit them?
